src/main/kotlin/no/nav/melosysskattehendelser/skatt/sigrun/SigrunRestConsumer.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.melosysskattehendelser.skatt.sigrun
+
+import mu.KotlinLogging
+import no.nav.melosysskattehendelser.skatt.Hendelse
+import no.nav.melosysskattehendelser.skatt.HendelseRequest
+import no.nav.melosysskattehendelser.skatt.SkatteHendelseConsumer
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.bodyToMono
+import java.time.LocalDate
+
+class SigrunRestConsumer(private val webClient: WebClient) : SkatteHendelseConsumer {
+    private val log = KotlinLogging.logger { }
+
+    override fun hentHendelseListe(request: HendelseRequest): List<Hendelse> =
+        webClient.get().uri("/api/skatteoppgjoer/hendelser")
+            .header("x-sekvensnummer-fra", request.seksvensnummerFra.toString())
+            .header("x-antall", request.antall.toString())
+            .header("x-bruk-aktoerid", request.brukAktoerId.toString())
+            .retrieve()
+            .bodyToMono<List<Hendelse>>()
+            .block() ?: emptyList()
+
+    override fun getConsumerId(): String = "sigrun-skatteoppgjoer-hendelser"
```

Comment:
Hva er dette consumerId som går igjen flere steder? ser det var i status tabellen også. Er det hvis vi skal utvide med annet api? Tenker vi kan vente med det isåfall, preoptimalisering osv

Reply:
Er for å ikke ha en direkte avhengihet til Sigrun fra SkatteHendelserFetcher. Vi vil jo isolere alt i sigrun.